src/MLOps.NET.Azure/Storage/StorageAccountMetaDataStore.cs

```diff
@@ -159,6 +164,18 @@ public List<IMetric> GetMetrics(Guid runId)
             return metrics;
         }
 
+        ///<inheritdoc/>
+        public IConfusionMatrixEntity GetConfusionMatrix(Guid runId)
```

Comment:
Just as we pass in our implementation of the ConfusionMatrix to LogConfusionMatrix, I think we should pass back our deserialized version here :)